bin/test-workflow/platform_login.sh

```diff
@@ -2,8 +2,48 @@
 
 set -euo pipefail
 IFS=$'\n\t'
+CONJUR_PLATFORM="${CONJUR_PLATFORM:-gke}"
+APP_PLATFORM="${APP_PLATFORM:-gke}"
+GCLOUD_PROJECT_NAME="${GCLOUD_PROJECT_NAME:-gke}"
+GCLOUD_ZONE="${GCLOUD_ZONE:-gke}"
+GCLOUD_CLUSTER_NAME="${GCLOUD_CLUSTER_NAME:-gke}"
+GCLOUD_SERVICE_KEY="${GCLOUD_SERVICE_KEY:-gke}"
+
+OPENSHIFT_URL="${OPENSHIFT_URL:-}"
+OPENSHIFT_USERNAME="${OPENSHIFT_USERNAME:-}"
+OPENSHIFT_PASSWORD="${OPENSHIFT_PASSWORD:-}"
+REGISTRY_URL="${REGISTRY_URL:-}"
+
+print_usage() {
+  echo "Usage:"
+  echo "    This script will log into the various platforms"
+  echo ""
+  echo "Syntax:"
+  echo "    $0 [Options]"
+  echo "    Options:"
+  echo "    -g                         GKE"
+  echo "    -h                         Show help"
+  echo "    -o                         Openshift"
+  echo "    -v <Openshift version>     The Openshift version"
+  echo "                               defaults to current"
+}
+
+function main() {
+  # Process command line options
+  local OPTIND
+  openshift_version="current"
+  while getopts ':ghov:' flag; do
+    case "${flag}" in
+      g) CONJUR_PLATFORM="gke" ;;
+      h) print_usage; exit 0 ;;
+      o) CONJUR_PLATFORM="oc";APP_PLATFORM="oc" ;;
```

Comment:
Nit: If both `-g` and `-o` are selected, the last one that is selected will be selected. Not a big deal, but if we want to make this more robust, you can disallow both being selected fairly easily by using a couple of local variables `gke_selected` and `oc_selected` in the case statement, and checking for both being selected after the while/case statements.

Reply:
updated with a check for that condition.